site2/docs/reference-pulsar-admin.md

```diff
@@ -2333,17 +2342,252 @@ Remove the message TTL for a topic.
 Usage
 ```bash
 $ pulsar-admin topics remove-message-ttl tenant/namespace/topic
-=======
+
+Options  
+```
+|Flag|Description|Default|
 |---|---|---|
 |`--enable`, `-e`|Enable message deduplication on the specified topic.|false|
 |`--disable`, `-d`|Disable message deduplication on the specified topic.|false|
 
-### `remove-deduplication`
-Remove a deduplication policy from a topic.
+### `get-retention`
+Get the retention policy that is applied to a topic.
 
 Usage
 ```bash
-$ pulsar-admin topics remove-deduplication tenant/namespace/topic
+$ pulsar-admin topics get-retention tenant/namespace/topic
+```
+
+### `set-retention`
+Set the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-retention tenant/namespace/topic
+```
+
+### `remove-retention`
+Remove the retention policy that is applied to a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-retention tenant/namespace/topic
+```
+
+### `get-dispatch-rate`
+Get message-dispatch-rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-dispatch-rate tenant/namespace/topic
+```
+
+### `set-dispatch-rate`
+Set message-dispatch-rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-dispatch-rate tenant/namespace/topic
+```
+
+### `remove-dispatch-rate`
+Remove message-dispatch-rate for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics remove-dispatch-rate tenant/namespace/topic
+```
+
+### `get-delayed-delivery`
+Get the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics get-delayed-delivery tenant/namespace/topic
+```
+
+### `set-delayed-delivery`
+Set the delayed delivery policy for a topic.
+
+Usage
+```bash
+$ pulsar-admin topics set-delayed-delivery tenant/namespace/topic
```

Comment:
should document the parameters defined here: https://github.com/apache/pulsar/blob/master/pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java#L1040-L1048